components/projects/form/fields/StatusField.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled';
+import { ChangeEvent, FC } from 'react';
+
+import Switch from '@/components/common/Switch';
+import Text from '@/components/common/Text';
+import { colors } from '@/styles/colors';
+import { textStyles } from '@/styles/typography';
+
+type Status = {
+  isAvailable: boolean;
+  isFounding: boolean;
+};
+
+interface StatusFieldProps {
+  className?: string;
+  value: Status;
+  onChange: (value: Status) => void;
+}
+
+const StatusField: FC<StatusFieldProps> = ({ className, value, onChange }) => {
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    onChange({
+      ...value,
+      [e.target.name]: e.target.checked,
+    });
+  };
+
+  return (
+    <StyledStatusField className={className}>
+      <StyledWrapper>
+        <StyledSubTitle>현재 이 서비스를 이용할 수 있나요?</StyledSubTitle>
+        <Switch name='isAvailable' checked={value.isAvailable} onChange={handleChange} />
+      </StyledWrapper>
+      <StyledWrapper>
+        <StyledSubTitle>현재 이 프로젝트로 창업을 진행하고 있나요?</StyledSubTitle>
+        <Switch name='isFounding' checked={value.isFounding} onChange={handleChange} />
+      </StyledWrapper>
+    </StyledStatusField>
+  );
+};
```

Comment:
p3; 지금 방식으로는 Status의 name에 대해 Type Safe 하지 않은데, 이런 식으로 바꾸는 것은 어때요?

```suggestion
const StatusField: FC<StatusFieldProps> = ({ className, value, onChange }) => {
  const handleChange = (name: keyof Status) => (e: ChangeEvent<HTMLInputElement>) => {
    onChange({
      ...value,
      [name]: e.target.checked,
    });
  };

  return (
    <StyledStatusField className={className}>
      <StyledWrapper>
        <StyledSubTitle>현재 이 서비스를 이용할 수 있나요?</StyledSubTitle>
        <Switch checked={value.isAvailable} onChange={handleChange('isAvailable')} />
      </StyledWrapper>
      <StyledWrapper>
        <StyledSubTitle>현재 이 프로젝트로 창업을 진행하고 있나요?</StyledSubTitle>
        <Switch checked={value.isFounding} onChange={handleChange('isFounding')} />
      </StyledWrapper>
    </StyledStatusField>
  );
};
```

Reply:
@Tekiter 좋네유 👍 반영하겠습니다~!